scripts/registrationStats.coffee

```diff
@@ -21,20 +21,36 @@ filter = (arr, func) ->
   filtered
 
 formatSummary = (data) ->
+  console.log(data)
```

Comment:
in coffeescript this can just be `console.log data`! also pls remove this from the commit